apps/condo/domains/banking/schema/BankTransaction.js

```diff
@@ -75,19 +83,10 @@ const BankTransaction = new GQLListSchema('BankTransaction', {
         purpose: {
             schemaDoc: 'Textual description of payment purpose in free form',
             type: Text,
+            defaultValue: true,
```

Comment:
Why default value of the field with `Text` type is become boolean?

Reply:
Oops, it is a bug